app/lib/fact_check_email_handler.rb

```diff
@@ -33,10 +29,9 @@ def process
       process_message(message)
       message.mark_for_delete = true
       yield(message) if block_given?
-    rescue StandardError => e
+    rescue StandardError
       message.mark_for_delete = false
       unprocessed_emails_count += 1
-      GovukError.notify(e)
```

Comment:
I think it would be good to log this exception so we can find out about it in Logit if we're needing to debug.